src/color.scss

```diff
@@ -3,7 +3,7 @@
 
 // ---------------------------------------------------------------------
 
-@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/alaska/SCSSVariables--alaska";
```

Comment:
minor nit - but all the other files use single quotes throughout our code. Why change to double quotes here?